magenta/music/melody_encoder_decoder.py

```diff
@@ -0,0 +1,428 @@
+
```

Comment:
What about some of those melody-specific input features in attention_rnn?  Are those gone now?